lib/wprof/conf_wprof.rb

```diff
@@ -0,0 +1,25 @@
+module WProf
+  class Config
+    DEFAULTS_CONFIGS = {
+      db_runtime: true,
+      reporter_type: 'LOGGER',
+      csv_type: 'SPLIT',
+      async: false,
+      httparty_methods_to_trace: %w[get put delete post].freeze,
```

Comment:
I woud also add the `PATCH` method

Reply:
Done! now 'patch' method is included for default into httparty methods.